.github/workflows/js-config.yml

```diff
@@ -2,12 +2,11 @@ name: centreon-js-config
 
 on:
   pull_request:
-    types: [closed]
+    types:
+      - closed
     paths:
       - ".github/workflows/js-config.yml"
       - "centreon/packages/js-config/**"
-    branches:
-      - develop
```

Comment:
Don't you want to target specific branches to trigger this workflow?

Reply:
Probably but I'm trying to figure out why the 'delivery-npm' job is triggered when the pull request is merged